druid/src/widget/flex.rs

```diff
@@ -585,34 +588,25 @@ impl<T: Data> Flex<T> {
 
 impl<T: Data> Widget<T> for Flex<T> {
     fn event(&mut self, ctx: &mut EventCtx, event: &Event, data: &mut T, env: &Env) {
-        for child in &mut self.children {
-            match child {
-                Child::Fixed(widget, _) | Child::Flex(widget, _, _) => {
-                    widget.event(ctx, event, data, env)
-                }
-                _ => {}
+        for child in self.children.iter_mut().map(|x| x.widget_mut()) {
+            if let Some(widget) = child {
```

Comment:
just a note, if you used `flat_map` instead of `map` you wouldn't need the `if let Some(..)` business.

Reply:
I got you one better, google tells me `filter_map` is even better!!!